config/config.md

```diff
@@ -176,7 +176,9 @@ must be the same with the number of given `--state-sync-ids`.
 
 #### `--partial-sync-primary-network` (string)
 
-Partial sync enables non-validators to optionally sync only the P-chain on the primary network.
+Partial sync enables nodes that are not primary validators to optionally sync only
+the P-chain on the primary network. Nodes that use this option can still track 
+Subnets. After the Etna upgrade, nodes that use this option can also validate L1s.
```

Comment:
```suggestion
Partial sync enables nodes that do not validate the [Primary Network](https://docs.avax.network/learn/primary-network) to sync only the P-Chain. Nodes that use this option can still track 
Subnets. After the Etna upgrade, nodes that use this option can also sync and/or validate L1s.
```

Reply:
phrasing seemed confusing, added a link to the doc that explains the primary network as well.

Reply:
If they track any Subnets they wouldn't only be syncing the P-Chain though 😅

Reply:
true 😵‍💫